foxglove recordings list`

**Description**
Running `foxglove recordings list` results in the following error:
```
❯ foxglove recordings list
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x732f82]

goroutine 1 [running]:
github.com/foxglove/foxglove-cli/foxglove/console.RecordingsResponse.Fields({{0xc000028a60, 0x10}, {0xc000312000, 0x46}, 0xeabefe, 0x36, {0xc00002c168, 0x18}, {0xc00002c180, 0x18}, ...})
        /home/runner/work/foxglove-cli/foxglove-cli/foxglove/console/api.go:232 +0x2c2
github.com/foxglove/foxglove-cli/foxglove/cmd.renderList[...]({0xb43ca0, 0xc000014018}, 0xa61e42, 0x5?, {0xa61e5b?, 0x5})
        /home/runner/work/foxglove-cli/foxglove-cli/foxglove/cmd/utils.go:100 +0x3de
github.com/foxglove/foxglove-cli/foxglove/cmd.newListRecordingsCommand.func1(0xc0001f6280?, {0xa618b3?, 0x0?, 0x0?})
        /home/runner/work/foxglove-cli/foxglove-cli/foxglove/cmd/recordings.go:38 +0x5b0
github.com/spf13/cobra.(*Command).execute(0xc0001f6280, {0xf13268, 0x0, 0x0})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:860 +0x663
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001b3680)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:974 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:902
github.com/foxglove/foxglove-cli/foxglove/cmd.Execute({0xb40f50, 0x7})
        /home/runner/work/foxglove-cli/foxglove-cli/foxglove/cmd/root.go:230 +0xd16
main.main()
        /home/runner/work/foxglove-cli/foxglove-cli/foxglove/main.go:11 +0x27
```

API key is loaded and working with other commands such as `foxglove devices list`. The API key I'm using has all options enabled. 


- Version:
```
❯ foxglove version
v1.0.12
```
- Platform:
Windows 11 running WSL with Ubuntu 22.04

**Steps To Reproduce**
<!--- Include the minimum possible code or example data to reproduce the problem. -->
Pretty simply running the `foxglove recordings list` command.


**Expected Behavior**
List of at least some of the recordings in the data platform.

--- a/foxglove/console/api.go
+++ b/foxglove/console/api.go
@@ -189,7 +189,7 @@ type RecordingsResponse struct {
 	End          string           `json:"end"`
 	ImportStatus string           `json:"importStatus"`
 	Site         SiteSummary      `json:"site"`
-	EdgeSite     *SiteSummary     `json:"edgeSite"`
+	EdgeSite     SiteSummary      `json:"edgeSite"`
 	Device       DeviceSummary    `json:"device"`
 	Metadata     []MetadataRecord `json:"metadata"`
 }

--- a/foxglove/console/api_test.go
+++ b/foxglove/console/api_test.go
@@ -1,0 +1,34 @@
+package console
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRecordingsFields(t *testing.T) {
+	t.Run("sets empty values for optional fields", func(t *testing.T) {
+		resp := RecordingsResponse{
+			ID:           "some-id-1",
+			Path:         "/file.mcap",
+			Size:         1024,
+			MessageCount: 1,
+			CreatedAt:    "2023-03-02T15:00:00.000Z",
+			ImportedAt:   "2023-03-03T15:00:00.000Z",
+			Start:        "2023-03-01T15:00:00.000Z",
+			End:          "2023-03-01T15:10:00.000Z",
+			ImportStatus: "pending",
+		}
+
+		edgeSiteIdx := -1
+		for i, header := range resp.Headers() {
+			if header == "Edge Site ID" {
+				edgeSiteIdx = i
+			}
+		}
+
+		assert.Greater(t, edgeSiteIdx, -1)
+		assert.Equal(t, "some-id-1", resp.Fields()[0])
+		assert.Equal(t, "", resp.Fields()[edgeSiteIdx])
+	})
+}

